src/components/Footer.tsx

```diff
@@ -1,15 +1,176 @@
-import React from 'react';
-import { Container } from 'styled-bootstrap-grid';
-import styled from 'styled-components';
+import React from "react";
+import { Container, Row, Col } from "styled-bootstrap-grid";
+import styled from "styled-components";
 
 const FooterContainer = styled.footer`
-  padding: 2rem 0;
-  margin-top: 10rem;
+  float: left;
+  width: 100%;
+  background: #eee;
```

Comment:
Hmm, can we add this style somewhere in `src/utils/styles.ts`?

Reply:
Which styles/line are you referring to? the background color?

Reply:
yep. can we add it to that file so it's in the theme? this feels like a `gray` color?